packages/common/src/content/_internal/computeProps.ts

```diff
@@ -17,5 +42,13 @@ export function computeProps(
   // thumbnail url
   content.thumbnailUrl = getItemThumbnailUrl(model.item, requestOptions, token);
 
+  if (!content.location) {
+    // build location if one does not exist based off of the boundary and the item's extent
+    content.location =
+      model.item.properties?.boundary === "none"
+        ? { type: "none" }
+        : deriveLocationFromItemExtent(model.item.extent);
+  }
+
```

Comment:
This logic will soon need to be updated for https://devtopia.esri.com/dc/hub/issues/7527#issuecomment-4269838